provider/cmd/tests/TestTasks.yml

```diff
@@ -2,6 +2,15 @@ version: "3"
 
 tasks:
   default:
+    desc: Run python integration tests without db tests.
```

Comment:
FYI you could use an alias here like this:
```yaml
tasks:
  test:
    aliases: [default]
    desc: Run tests
    cmds:
      - echo "foo"
```

What this does is make the command named and listed in `task --list`, while still being available as the default `task`